addons/website_slides/static/tests/tours/slides_tour_tools.js

```diff
@@ -324,6 +324,44 @@ var addNewCourseTag = function (courseTagName, backend) {
 }];
 };
 
+/**
+ * Adds a new quiz to the given section name.
+ */
+const addQuizToSection = (sectionName, quizName, backend) => {
+    const prefix = backend ? ':iframe ' : '';
+    return [
+{
+    content: 'eLearning: add content to section',
+    trigger: prefix + 'div.o_wslides_slide_list_category_header:contains("' + sectionName + '") a:contains("Add Content")',
```

Comment:
Can we use what is in `addPdfToSection`?
```suggestion
    trigger: `${prefix}div.o_wslides_slide_list_category_header:contains("${sectionName}") a:contains("Add Content")`,
```

Actually, we could add a helper function to get this common step in all these functions.

Reply:
Hello @flch-odoo :wave: ,
Thanks for the validation :slightly_smiling_face: ,

I have implemented most of your suggestions and realized that I could optimize the process of adding questions to the quiz. I have improved it.